users/contrib.rst

```diff
@@ -77,9 +77,11 @@ Snap
 
 On any of the `Linux distributions that support snaps <https://snapcraft.io/docs/core/install>`_: ::
 
-   $ snap install syncthing --candidate
-   
-(Note that this is an experimental and unstable release, at the moment)
+   $ snap install syncthing
+
+If you want to help testing the upcoming release, you can install the snap from the candidate channel:
```

Comment:
I would also mention the positive aspects, something like "and get the newest features".